packages/react/test/evaluation.spec.tsx

```diff
@@ -577,5 +580,76 @@ describe('evaluation', () => {
         );
       });
     });
+
+    describe('HookFlagQuery',  () => {
+      it('should return errors correctly', () => {
+        const errorMessage = 'no flag found with key i-dont-exist';
+        const details: EvaluationDetails<boolean> = {
+          flagKey: 'i-dont-exist',
+          flagMetadata: {},
+          errorCode: ErrorCode.FLAG_NOT_FOUND,
+          errorMessage,
+          value: true,
+        };
+
+        let hookFlagQuery = new HookFlagQuery(details);
+        expect(hookFlagQuery.isError).toEqual(true);
+        expect(hookFlagQuery.errorCode).toEqual(ErrorCode.FLAG_NOT_FOUND);
+        expect(hookFlagQuery.errorMessage).toEqual(errorMessage);
+
+        details.errorCode = undefined;
+        details.reason = StandardResolutionReasons.ERROR;
+        hookFlagQuery = new HookFlagQuery(details);
+        expect(hookFlagQuery.isError).toEqual(true);
+      });
+
+      it('should return flag metadata', () => {
+        const flagMetadata = {
+          'ping': 'pong'
+        };
+        const details: EvaluationDetails<boolean> = {
+          flagKey: 'with-metadata',
+          flagMetadata,
+          value: true,
+        };
+        const hookFlagQuery = new HookFlagQuery(details);
+        expect(hookFlagQuery.flagMetadata).toEqual(flagMetadata);
```

Comment:
`expect.ObjectContaining` is a less-brittle way of testing this sort of thing, if a new prop is added it won't fail... on the other hand sometimes that's good because it forces you to update the test.

Reply:
Great suggestion I didn't think of that - I'll amend now!

Reply:
Added